charts/router/templates/_helpers.tmpl

```diff
@@ -0,0 +1,10 @@
+{{/*
+Set apiVersion based on Kubernetes version
+*/}}
+{{- define "rbacApiVersion" -}}
```

Comment:
If observe Go's established naming conventions, this should be `rbacAPIVersion`. See doc [here](https://github.com/golang/go/wiki/CodeReviewComments#interfaces).